changelog/unreleased/group-without-gidnumber.md

```diff
@@ -0,0 +1,6 @@
+Change: Allow LDAP groups to have no gidNumber
+
+Similar to the user-provider allow a group to have no gidNumber. Assign
+a default in that case.
+
+https://github.com/cs3org/reva/pull/xxxx
```

Comment:
:rofl:

Reply:
For got to push the change that actuall inserts the correct number ;-)